### DISCUSSION TITLE
Cómo indexar una web en Google rápido

### DISCUSSION THREAD

**Discussion by giscus** [zkreations/data#34] [Announcements]
# Cómo indexar una web en Google rápido

Aprende método para indexar cualquier sitio web o blog en Google fácilmente de forma rápida.

https://www.zkreations.com/2024/04/como-indexar-en-google-rapidamente.html

**Comment by danieIabel** [comment]
Este artículo se centra en Google, pero a futuro cuando Google sea compatible con el protocolo **IndexNow** publicaré uno nuevo, ya que tengo planes de trabajar en un proyecto que sirva para enviar enlaces a todos los buscadores a la vez.

**Comment by danieIabel** [comment]
Ha llegado a mi un método para poder omitir por completo la instalación de localwp. Me pondré a desarrollarlo y cuando lo tenga listo actualizaré el artículo.

**Reply by danieIabel** [reply]
He actualizado el tutorial para enviar los enlaces a Google. Ya no se requiere localwp ni instalar nada.

**Comment by llusan** [comment]
¿Puedo usar el mismo archivo json para otros dominios?
Es decir, para un segundo, tercero o posteriores dominio... tendría que saltarme al paso de ir a Google Search Console, ejecutar el Google Collab e ingresar los links. ¿Sería así?

Gracias por compartir este método, espero que sea seguro 🙈.

**Reply by danieIabel** [reply]
Buenos dias amigo. Asi es, no es necesario crear mas claves de acceso, la misma clave sirve para todos los dominios que tengas en Search Console. Solo debes asegurarte de que todos los dominios que tengas en Search Console tengan agregada la cuenta de servicio. Es decir que tendrás que repetir únicamente el paso "Agregar cuenta de servicio en Google Search Console" una vez por cada dominio que hayas agregado en Search Console.

El método es seguro, sin embargo te recomiendo que solo envies contenido de calidad al buscador. Si envías enlaces que no lo son, lo mas probable es que Google los termine borrando después de un tiempo y eso te puede perjudicar en SEO.

**Reply by llusan** [reply]
Gracias nuevamente,
Llevo años intentando indexar fallidamente. Recuerdo que apenas tenía 2 links que no se indexaban y cuando por fin lo hizo, aparecieron 10 nuevos, volvía a solicitar y aparecieron 20, 30... y ahora son 60 🙈.
¡Gracias!

**Comment by ivisjose** [comment]
pregunta amigo con este método ya  no es necesario ir a search consola, a inspección de urls y luedo dar en solicitar indexación? solo con ejecutar la la API de Indexing seria todo

**Reply by danieIabel** [reply]
Buenas tardes. Asi es amigo, con tan solo enviar mediante la api los enlaces serán indexados. Search Console los detectará indexados la proxima vez que los rastree. También puedes comprobar la indexación usando el parámetro "site:www.tusitio.com" en Google.

**Nota:** Recuerda que si envías enlaces de baja calidad o que vayan en contra de las políticas de google, aunque sí aparecerán en el buscador, probablemente serán retirados, por lo cual siempre envía contenido de calidad.

**Reply by ivisjose** [reply]
Gracias amigo, pero siempre tengo que enviar un sitemap en search console o no es necesario ya agregarlo

**Comment by ivisjose** [comment]
Hola intento ingresar "Google Colab", pero me da el error de (Lo sentimos. No puedes acceder a este documento porque infringe las Condiciones del Servicio.)  :(

**Reply by danieIabel** [reply]
Buenos dia amigo. He revisado y Google lo ha desactivado para "compartir" pero si sigue en mi cuenta el proyecto. La alternativa a esto sería alojar el proyecto nosotros mismos. Estoy evaluando la forma de hacerlo de tal manera que no me salga muy caro.

**Comment by estebanpaglia** [comment]
Hola amigo, gracias por compartir tan valiosa información. Tengo problemas para indexar en blogger y encontré tu post en la web. Hice todos los pasos para crear, cómo se describe en los videos, pero tengo una duda: en la API debo agregar mi blog? o sólo colocando el usuario en GSC ya está.?

Envié varias urls mediante la API que se encuentra aquí en tu web (72hs aprox) y aún no tengo novedades. El mensaje fué de "200". ¿Creés que puede demorar más de una semana para ver resultados? Desde ya, te agradezco tu tiempo. Saludos!

**Reply by danieIabel** [reply]
Buenas tardes amigo. Si usas Blogger te recomiendo el método 1 (facil) y tus enlaces deberian aparecer rápidamente. Si usas la api cambia el tipo de solicitud a "Status" para verificar si realmente Google está entendiendo tu solicitud, si no lo hace (obtienes un error) es por eso que tus enlaces no han aparecido.

**Reply by estebanpaglia** [reply]
Gracias por tu respuesta. Yo las había enviado con "URL UDPDATE", había recibido codigo "200". Voy a probar cómo dices. Muchísimas gracias!

**Comment by ivisjose** [comment]
hola amigos, hola Daniel, les comento que ya intente de todo para lograr indexar mi blog pero solo se ha indexado la pagina principal. he enviado enlaces por la la opcion de Api. todo bien recibo como respuesta 200 tambien en status pero en search console me errores de:

- Error de redirección
-Página con redirección
-Página alternativa con etiqueta canónica adecuada

y no entiendo que mas puedo hacer, estara algo mal ala hora de configurar dominio de goddady con blogger? o faltara algún metadato? en mi blog

comentarles que tengo hace tiempo tratando :P ya le di tiempo para que aparecieran y nada espero me ayuden o me den algun consejo  de ante mano muchas gracias

**Reply by danieIabel** [reply]
Que tal amigo, hace unas semanas actualice este mismo artículo, con una opción rápida y fácil para Blogger que puede ver casi al inicio, no requiere la API. ¿Probo ese método? Ya que algunos usuarios han estado teniendo problemas con la API cuando se envían muchas urls, otros simplemente no les funciona. Pruebe con el primer método que está en este mismo artículo, compruebe primero con pocas urls (por ejemplo 10) y luego verifique si están indexadas.

**Reply by ivisjose** [reply]
hola grax por responder, por cierto he estado utilizando ese método hasta que volvio en linea la opcion API que nuevamente publicaste pero aun a si sigue sin indexar solo me da error de redirección y me da mensaje de "X" url econtradas no indexadas y 1 idenxada que es la principal uso goddady como dominio y la plantilla con licencia que uso de bison no le he tocado nada mas que las configuraciones de colores y fuente :P

**Reply by danieIabel** [reply]
Es difícil que le pueda indicar porque no son aceptadas las urls amigo. Google puede tener sus propios motivos o problemas. La única pista que tengo es que Google no desea que se use la api para enviar webs que considera Spam. Es por eso que aun recibiendo un código 200 si la solicitud no la tiene Google entonces no pierda el tiempo y pruebe otro método. En todo momento usted puede comprobar si la solicitud la tiene Google usando "URL_STATUS" para comprobar si hay una solicitud con esas urls en cola. Si recibe un código 404 quiere decir que la solicitud que envió para esas urls no fue tomada en cuenta por google, aun haya sido enviada correctamente.
